test/images/image-util.sh

```diff
@@ -142,11 +142,16 @@ push() {
 bin() {
   for SRC in $@;
   do
+  if [[ "${ARCH}" == "arm" ]]; then
+    arch_prefix="GOARM=${GOARM:-7} GOARCH=arm"
+  else
+    arch_prefix="GOARCH=${ARCH}"
+  fi
   docker run --rm -it -v "${TARGET}:${TARGET}:Z" -v "${KUBE_ROOT}":/go/src/k8s.io/kubernetes:Z \
         golang:"${GOLANG_VERSION}" \
         /bin/bash -c "\
                 cd /go/src/k8s.io/kubernetes/test/images/${SRC_DIR} && \
-                CGO_ENABLED=0 GOARM=${GOARM} GOARCH=${ARCH} go build -a -installsuffix cgo --ldflags '-w' -o ${TARGET}/${SRC} ./$(dirname "${SRC}")"
+                CGO_ENABLED=0 ${arch_prefix} go build -a -installsuffix cgo --ldflags '-w' -o ${TARGET}/${SRC} ./$(dirname "${SRC}")"
```

Comment:
```suggestion
                CGO_ENABLED=0 GOARM=${GOARM:-7} GOARCH=${ARCH} go build -a -installsuffix cgo --ldflags '-w' -o ${TARGET}/${SRC} ./$(dirname "${SRC}")"
```

Reply:
though probably the GOARM defaulting should be fixed wherever we're actually setting / exporting it instead.

Reply:
@BenTheElder , thanks for the review,  I don't think so, cause GOARM is only needed when the GOARCH is "arm". 

The change below will make the GOARM always been set regardless it is `amd64`, `arm` or whatever arch it is.

`GOARM=${GOARM:-7}`

Reply:
this appears to actually be the true case anyhow https://github.com/kubernetes/kubernetes/issues/79689#issuecomment-507936520

these are usually built from `test/images` which sets GOARM=7, I'm not sure we need this PR vs some better docs. https://github.com/kubernetes/kubernetes/issues/79689#issuecomment-507936520